src/app/core/services/api.service.ts

```diff
@@ -244,4 +245,15 @@ export class ApiService {
 
         return this.http.get(baseUrl, { headers, params });
     }
+
+    getRnaDistributionData(): Observable<RnaDistribution[]> {
+        const headers = new HttpHeaders({
```

Comment:
This could be a constant that is used across the app (I'd guess that the same headers are used for the majority of the service calls).  Named something like JSON_RESPONSE_NO_CACHE_HEADERS

Reply:
Added `const` on top of the file, I don't think it is used anywhere else